internal/ent/schema/workflowdefinition.go

```diff
@@ -95,6 +98,9 @@ func (WorkflowDefinition) Edges() []ent.Edge {
 			name:       "target_tags",
 			edgeSchema: TagDefinition{},
 			comment:    "Tags this workflow targets for scoping",
+			annotations: []schema.Annotation{
+				accessmap.EdgeNoAuthCheck(),
```

Comment:
This seems incorrect - you should have view access to the tag to be able to add it? 

```suggestion
				accessmap.EdgeViewCheck(TagDefinition{}.Name()),
```